clinica/iotools/converters/genfi_to_bids/genfi_to_bids_utils.py

```diff
@@ -75,6 +82,20 @@ def filter_dicoms(df: DataFrame) -> DataFrame:
     return df
 
 
+def _handle_series_description(x: str) -> str:
+    series_description = pdcm.dcmread(x).SeriesDescription
+    if any([modality in series_description for modality in ["t1", "t2", "T1", "T2"]]):
```

Comment:
Does it happen in practice to have mixtures of upper and lower case ?

This feels a little weird when reading it. I would expect the value(s) in `series_description` to be made lower case and then compared to "t1" and/or "t2". Does this make sense ?